### PR TITLE
Increase testing

### PR DESCRIPTION
**Pull request by aaronchongth**
## What's new

* Correct `codecov.yml`
* Increase testing

## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

**Comment by aaronchongth** [comment]
To keep the commits small, I'll continue with more testing on a subsequent PR

File: codecov.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ coverage:
   status:
     project: false
     patch: false
+ignore:
+  - packages/rmf-dashboard-framework/examples
+  - "**/*.js"
+  - "**/*.stories.tsx"
+  - "**/index.ts"
```
Comment (koonpeng) [inline codecov.yml:9]: `index.ts` shouldn't be ignored.

File: packages/rmf-dashboard-framework/src/components/map/map.tsx [hunk 2]
```diff
@@ -532,6 +533,7 @@ export const Map = styled((props: MapProps) => {
           camera.updateProjectionMatrix();
         }}
         orthographic={true}
+        resize={{ polyfill: ResizeObserver }}
```
Comment (koonpeng) [inline packages/rmf-dashboard-framework/src/components/map/map.tsx:536]: Looking at the browser support, I don't think there is a need for us to polyfill.

**Review by koonpeng — CHANGES_REQUESTED** [review]

File: packages/rmf-dashboard-framework/src/components/map/map.tsx (hunk 2 above)
Comment (aaronchongth) [reply packages/rmf-dashboard-framework/src/components/map/map.tsx]: Yup, that was added only for testing, I've found a solution for that on https://github.com/ZeeCoder/use-resize-observer/issues/40#issuecomment-644536259 to use a mock resize observer

https://github.com/open-rmf/rmf-web/pull/1036/commits/3affe31c348d916d8f6858051c45479455a9caf6

File: codecov.yml (hunk 1 above)
Comment (aaronchongth) [reply codecov.yml]: Thanks for catching that, I accidentally left that in after trying some stuff

https://github.com/open-rmf/rmf-web/pull/1036/commits/3affe31c348d916d8f6858051c45479455a9caf6

**Review by koonpeng — APPROVED** [review]

**Pull request merged**